### ISSUE TITLE
[Discussion] [Request] Rename `service` => `handlers` in options

### ISSUE THREAD

**Issue by maslennikov** [seriousme/fastify-openapi-glue#489]
Hello and thank you for this plugin! I am its recent user, and what I noticed is somewhat unfortunate naming of options fields. Both `service` and `securityHandlers` serve their sole purpose of defining fastify http route handlers. In other MVC-ish frameworks they could also be called controllers, while services are some injected or global entities comprising business logic on top of data models.

My proposal would be to rename `service` to `handlers`, so that naming stays consistent:
```js
const options = {
    handlers: ...,
    securityHandlers: ...
    ...rest
}
```

What do you think?

**Comment by seriousme** [comment]
Hi Alexey,

I think that, with hindsight, you might be right :-)
Unfortunately it is rather a breaking change to rename options as  nearly every current user would need to change their code :-(
If you really feel bothered by the name you can provide your own `operationResolver`  ;-)

Kind regards,
Hans
ps. `service` is also not too bad  of a name as the initial idea was that you would have a service that you would front with an fastifiy api layer ;-)

**Comment by seriousme** [comment]
Hi,

Looking at it again I think that:

```javascript
const options = {
    serviceHandlers: ...,
    securityHandlers: ...
    ...rest
}
```
Might even be more appropriate  as `securityHandlers` are also  a type of handler ;-)

We could add an alias of `serviceHandlers` to `service` e.g. with a deprecated note.
What do you think ?

Kind regards,
Hans

**Comment by maslennikov** [comment]
Hi @seriousme, I think that adding an alias with a deprecation note is a a good idea. 

Also, regarding `serviceHandlers` naming I think it is even more appropriate, since they differ from native fastify handlers in regard of incompatible `this` context, so the name will at least give a slight hint that it is an entity of some other nature.

**Comment by seriousme** [comment]
FYI: released as 4.3.0 on NPM

Thanks for the feedback !
Hans
